assets/setup-wizard/index.js

```diff
@@ -63,23 +74,31 @@ const SenseiSetupWizardPage = () => {
 				{ __( 'Error details:', 'sensei-lms' ) } { error.message }
 			</Notice>
 		);
-	}
-
-	return (
-		<QueryStringRouter
-			paramName={ PARAM_NAME }
-			defaultRoute={ navigationSteps.find( ( step ) => step.isNext ).key }
```

Comment:
Maybe later we will want to add something for the user to return to the step where the user stopped (in case they close the setup wizard before completing it), but I think we can make something much simpler (probably local storage is enough instead of a big customization around completed steps).